server/handlers.go

```diff
@@ -160,14 +174,44 @@ func (svr *Server) handleGetEigenDADispersalBackend(w http.ResponseWriter, _ *ht
 // =================================================================================================
 
 // handlePostStdCommitment handles the POST request for std commitments.
```

Comment:
comment is misplaced

Reply:
04dba8c7e4368cfd65fdfc4b9bb95f480a8ec008